sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/ServiceBusAdministrationAsyncClient.java

```diff
@@ -2052,17 +2091,32 @@
             return monoError(logger, new NullPointerException("'context' cannot be null."));
         }
 
+        final HttpHeaders supplementaryAuthHeaders = new HttpHeaders();
+        Context additionalContext = context.addData(AZ_TRACING_NAMESPACE_KEY, SERVICE_BUS_TRACING_NAMESPACE_VALUE);
+        if (!CoreUtils.isNullOrEmpty(queue.getForwardTo())) {
+            addAdditionalAuthHeader(SERVICE_BUS_SUPPLEMENTARY_AUTHORIZATION_HEADER_NAME, supplementaryAuthHeaders);
+            queue.setForwardTo(String.format("https://%s/%s", managementClient.getEndpoint(),
```

Comment:
Use a string constant?

Reply:
Q: also should we be using a UriBuilder instead of concatenated strings?

Reply:
went with the UrlBuilder in azure core. It is an overkill here, but this should cover us in case of any future expansions